### PR TITLE
Update schema and bindings for VEC 2.1.0 (vec-v2x)

### PR DESCRIPTION
**Pull request by beckerjohannes**
## Pull Request

- [X] I have checked for similar PRs.
- [X] I have read the [contributing guidelines](https://github.com/4Soft-de/harness-model/blob/develop/.github/CONTRIBUTING.md).

### Changes

- [X] Code
- [ ] Documentation
- [ ] Other: 

### Description

Updated library `vec-v2x` to schema version VEC 2.1.0 which should be compatible to 2.0.2 and purly addiditv.

### CODE REVIEW

**Comment by RainerGanss** [comment]
doesn´t build.

**Comment by beckerjohannes** [comment]
@RainerGanss @takebene the SonarCloud quality gate makes not sense for a PR like that.

File: scripts/relocate-doc-in-xsd.xsl [hunk 1]
```diff
@@ -9,7 +9,7 @@
     <xsl:template match="xs:documentation">
         <xs:documentation>
             <xsl:text disable-output-escaping="yes">&lt;![CDATA[</xsl:text>
-            <xsl:copy-of select="*"/>
+            <xsl:copy-of select="*|text()"/>
```
Comment (RainerGanss) [inline scripts/relocate-doc-in-xsd.xsl:12]: what does this change?

Comment (beckerjohannes) [reply scripts/relocate-doc-in-xsd.xsl:12]: This: 
image
The old version dropped embedded documentation that was just plain text and not html formatted like this one:

```xml
<xs:complexType name="AbstractLocalizedString" abstract="true">
   <xs:annotation>
       <xs:documentation xml:lang="en">Abstract super-class for Localized text values.</xs:documentation>
   </xs:annotation>
....
<xs:sequence>
```

**Review by RainerGanss — APPROVED** [review]

**Pull request merged**